tool/tsh/tsh.go

```diff
@@ -1187,16 +1187,19 @@ func serializeVersion(format string, proxyVersion string) (string, error) {
 // onPlay is used to interact with recorded sessions.
 // It has several modes:
 //
-// 1. If --format is "pty" (the default), then the recorded
-//    session is played back in the user's terminal.
-// 2. Otherwise, `tsh play` is used to export a session from the
-//    binary protobuf format into YAML or JSON.
+//  1. If --format is "pty" (the default), then the recorded
+//     session is played back in the user's terminal.
+//  2. Otherwise, `tsh play` is used to export a session from the
+//     binary protobuf format into YAML or JSON.
 //
 // Each of these modes has two subcases:
 // a) --session-id ends with ".tar" - tsh operates on a local file
-//    containing a previously downloaded session
+//
```

Comment:
Why do we need an empty line here?

Reply:
This is related to Go 1.19's [doc comments](https://tip.golang.org/doc/go1.19#go-doc).  

I think `gopls` does this formatting automatically. 

It does indeed look incorrect though. I'll see if I can reformat it in a way where this doesn't happen.